books/free-programming-books-subjects.md

```diff
@@ -108,6 +108,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Data Structures and Algorithms Tutorial](https://www.tutorialspoint.com/data_structures_algorithms/) - Tutorials Point (HTML, PDF)
 * [Data Structures Succinctly Part 1, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/datastructurespart1) (PDF, Kindle) (email address *requested*, not required)
 * [Data Structures Succinctly Part 2, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/datastructurespart2) (PDF, Kindle) (email address *requested*, not required)
+* [Data Structures With C](https://www.fresources.tech/dtu/coe/Data%20Structures/file?fileId=https%3A%2F%2Fdata-storage.sgp1.digitaloceanspaces.com%2FDTU%2FDS%2520with%2520C%2520-%2520By%2520Schaum.pdf) - Seymour Lipschutz (PDF)
```

Comment:
Is this a free book?

Reply:
Yes it is.
The link does not require any email either.